docusaurus/docs/dev-docs/preview.md

```diff
@@ -164,13 +164,19 @@ const getPreviewPathname = (uid, { locale, document }): string => {
       }
       return `/blog/${slug}`; // Individual article page
     }
+    default: {
+      return null;
+    }
   }
-  return "/"; // Default fallback route
 };
 
 // … main export (see step 3)
 ```
 
+:::note
+Some content types don't need to have a preview if it doesn't make sense, hence the default case returning `null`. A Global single type with some site metadata, for example, will not have a matching frontend page. In these cases, the handler function should return `null`, and the preview UI will not be shown in the URL. This is how you enable/disable preview per content type.
```

Comment:
I know this is quoting me but I think I got confused on this part:

>and the preview UI will not be shown in the URL

should probably be

>and the preview UI will not be shown in the admin